### DISCUSSION TITLE
Approval for github educational developer program.

### DISCUSSION THREAD

**Discussion by saqlainixii** [community/community#19860] [GitHub Education]
Hey ! 
Can someone tell me, how much does github team takes to approve Education Developer program.

I applied for github education developer program on 27june but my approval request is still pending

**Comment by sinkaroid** [comment]
Yes, The estimated date that stated in your email can be misleading,  

It is because of the amount of applications they receive. They need to manually select and approve it so that the service is not abused.
Also you can fire them an email so they can speed up the process.  

Have you try to contact them again? [here](https://support.github.com/request/account?tags=dotcom-footer%2Chubberfy_account)
